src/icon/musicIcon.tsx

```diff
@@ -4,14 +4,14 @@ import isEmpty from 'lodash.isempty'
 
 import { color } from '_utils/branding'
 
-export enum MusicType {
+enum statusList {
```

Comment:
- Theses statuses are used by the 3 icons. Shouldn't it be put somewhere else?
- Are theses constants available outside? Like `MusicIcon.status.DEFAULT` or something?

Reply:
Added a "_utils/icon/status" file with the status and the "Barred" element so that the 3 icons import them from this one source